benchopt/plotting/generate_html.py

```diff
@@ -495,7 +495,7 @@ def plot_benchmark_html_all(patterns=(), benchmarks=(), root=None,
         fnames = []
         for p in patterns:
             fnames += (benchmark / 'outputs').glob(f"{p}.csv")
-        fnames = sorted(set(fnames))
+        fnames = sorted(set(fnames))[::-1]
```

Comment:
The elements are not sorted here but in the JS table because of jquery/aria anyway, so this can be removed